x/mint/testutil/app.yaml

```diff
@@ -5,7 +5,7 @@ modules:
 
       app_name: MintApp
 
-      begin_blockers: [staking, auth, bank, mint, genutil, params]
+      begin_blockers: [mint, staking, auth, bank, genutil, params]
```

Comment:
Why this change?

Reply:
It was just for having consistency between the simapp yaml and the module app.yaml